release-notes.md

```diff
@@ -366,6 +374,70 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v46.0
+
+This release has been tracked in [v46.0
+group](https://github.com/orgs/cloud-hypervisor/projects/6/views/4?filterQuery=release%3A%22Release+46%22)
+of our [roadmap project](https://github.com/orgs/cloud-hypervisor/projects/6/).
+
+### Advisory Locks Support for Disk Image Files
+
+Now advisory locks support is added and enabled by default to prevent
+the same disk image file (that backs up a `virtio-blk` device) from being
```

Comment:
Maybe drop the "up" - since we're talking about user visible changes I would refer to it as as "--block".

Reply:
I'd drop that "advisory" from here, it is an implementation detail. From a user perspective, locking is mandatory.


How about:
_Cloud Hypervisor now enforces file-level locking for disk backing files (--block devices). This ensures that only a single Cloud Hypervisor instance can obtain write access to a given disk image at any time, preventing misconfiguration and avoiding potential data corruption.

When a disk is marked as read-only, multiple instances can acquire a shared (read) lock concurrently. However, exclusive (write) access is denied until all shared locks have been released.

This locking mechanism is fully compatible with state save/resume and live migration workflows. Disk devices backed by vhost_user are not subject to this locking policy. _

Reply:
@rbradford I believe you meant to say `--disk` from our CLI. @phip1611 Thanks for the inputs. Let me revise what I wrote.